SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -32,6 +33,19 @@ namespace
 	};
 
 	UEStream Stream;
+
+#if TRACE_LIB_ACTIVE
+	improbable::trace::SpanContext ReadSpanContext(const void* TraceBytes, size_t TraceSize, const void* SpanBytes, size_t SpanSize)
```

Comment:
No need to pass the trace size in here, lets just use sizeof(TraceID) etc. internally.